src/actions/EditNew.ts

```diff
@@ -150,9 +150,15 @@ class EditNew implements Action {
       selections: state.thatRanges.map(toSelection),
     };
 
+    const cursorInfos = state.cursorRanges
+      .map((range, index) => ({ range, index }))
+      .filter(({ range }) => range != null);
+
+    const cursorIndices = cursorInfos.map(({ index }) => index);
+
     const cursorSelections = {
-      selections: state.cursorRanges.map(toSelection),
-      rangeBehavior: DecorationRangeBehavior.OpenOpen,
+      selections: cursorInfos.map(({ range }) => toSelection(range!)),
+      rangeBehavior: DecorationRangeBehavior.ClosedClosed,
```

Comment:
It determines what happens if text is inserted adjacent to a selection.  If it's open on that side, the selection will expand to include the new text, otherwise it won't.  If for some reason a delimiter insertion action happened adjacent to one of the cursors from a command action, the command action cursor would expand, which we don't want.  It's unlikely we'd ever hit this case, but 🤷‍♂️